containerize/Program.cs

```diff
@@ -1,90 +1,112 @@
 ﻿using System.CommandLine;
 using Microsoft.NET.Build.Containers;
 using System.Text.Json;
+using System.CommandLine.Parsing;
 
-var fileOption = new Argument<DirectoryInfo>(
-    name: "folder",
-    description: "The folder to pack.")
+var publishDirectoryArg = new Argument<DirectoryInfo>(
+    name: "PublishDirectory",
+    description: "The directory for the build outputs to be published.")
     .LegalFilePathsOnly().ExistingOnly();
 
-Option<string> registryUri = new(
-    name: "--registry",
-    description: "Location of the registry to push to.",
-    getDefaultValue: () => "localhost:5010");
-
-Option<string> baseImageName = new(
-    name: "--base",
-    description: "Base image name.",
-    getDefaultValue: () => "dotnet/runtime");
-
-Option<string> baseImageTag = new(
-    name: "--baseTag",
-    description: "Base image tag.",
-    getDefaultValue: () => $"{System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription[5]}.0");
-
-Option<string[]> entrypoint = new(
-    name: "--entrypoint",
-    description: "Entrypoint application command.");
-
-Option<string> imageName = new(
-    name: "--name",
-    description: "Name of the new image.");
+var baseRegistryOpt = new Option<string>(
+    name: "--baseregistry",
+    description: "The registry to use for the base image.")
+{
+    IsRequired = true
+};
 
-var imageTag = new Option<string>("--tag", description: "Tag of the new image.", getDefaultValue: () => "latest");
+var baseImageNameOpt = new Option<string>(
+    name: "--baseimagename",
+    description: "The base image to pull.")
+{
+    IsRequired = true
+};
 
-var workingDir = new Option<string>("--working-dir", description: "The working directory of the application", getDefaultValue: () => "/app");
+// // Add validator here
```

Comment:
?

Reply:
Skipping this for now. This app will be called after `ParseContainerProperties`, which should do a lot of the validation. If we need to make this a better user-facing app then we can go for validating everything else.